Makefile

```diff
@@ -340,7 +344,7 @@ create-cluster: $(CLUSTERCTL) ## Create a development Kubernetes cluster on AWS
 	# Create a worker node with MachineDeployment.
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		create -f examples/_out/machinedeployment.yaml
+		apply -f examples/_out/machinedeployment.yaml
```

Comment:
Why apply here? I'd worry that we could potentially mutate something existing vs creating from scratch here.